### ISSUE TITLE
Move integration test to Zopkio

### ISSUE THREAD

**Issue by sriramsub** [linkedin/ambry#304]
Zopkio is a well tested distributed testing framework and would be ideal for writing integration tests. We should slowly start moving our integration tests into that. A good first step would be to setup Zopkio and write a very simple test - 

1. Start server
2. Start frontend
3. Put blob and get blob and verify

Once this works, we can opportunistically move existing test cases to that.

More information about Zopkio can be found here - https://github.com/linkedin/Zopkio

**Comment by vgkholla** [comment]
This could be newbie?

**Comment by sriramsub** [comment]
nope...newbie should typically spend time understanding a small portion of Ambry and fix the issue end to end in one week.
